### PR TITLE
Some bloodsucker ability nerfs

### PR DESCRIPTION
**Pull request by Arturlang**
## About The Pull Request
Nerfs a few too-often used bloodsucker abilities by increasing blood costs and cooldowns slightly, also makes using haste disable fortitude if it is active.
brawn bloodcost 8->10, cooldown 9->12 seconds
fortitude cooldown 8-20 seconds, also you will not be able to heal any brute or burn damage while it is active
The scaling of the damage resist is also halved, no longer is maximum resistance of 0.3 for brute, 0.5 for burn achieved at level 4, now it is reached at level 8, changing by 0.05 per ability level
haste bloodcost 6->9, cooldown 12->15 seconds

## Why It's Good For The Game
Brawn, haste, fortitude, can make a bloodsucker really really difficult to kill, *especially* when it's in the hands of a skilled player, the purpose of this PR is to make it harder to combo abilities constantly, by increasing costs and cooldowns, and removing a combo that removes the demerits of fortitude, using haste while you have fortitude, that forces you to walk.
A big issue with cooldowns and their abilties is that the ability cooldowns get reduced per ability level, so increasesing these cooldowns is important.
As for the damage resistance scaling change, reaching level 8 is fairly likely with the current changes, and maximum damage resistance is really tough to deal with, 4 is quite easy to reach within a average shift.

## Changelog

:cl:
balance: brawn costs 10 blood instead of 8, and has a cooldown of 12 seconds, instead of 9
balance: fortitude now has a cooldown of 20 seconds, from 8, also you will not be able to heal any brute or burn damage while it is active, the scaling of the damage resist is also halved, no longer is maximum resistance of 0.3 for brute, 0.5 for burn achieved at level 4, now it is reached at level 8, changing by 0.05 per ability level
balance: haste costs 9 blood instead of 6, and has a cooldown of 15 seconds from 12
/:cl:

### CODE REVIEW

**Comment by Arturlang** [comment]
feedback is welcome and nothing is set in stone

**Comment by Moostard** [comment]
They wouldnt be hard to kill if fortitude did not give them burn damage resistance, which they are normally weak to.
Iirc it was added in one of previous PRs.
Any high level vampire can become an immune to stuns bullet sponge. Add the burn resistance and a good laser armor on top and even lasers have trouble dealing enough damage (this is what some players did).

I believe that with fortitude no longer giving burn resistance like it was originally, there is no way a bloodsucker can become close to unstoppable.

**Comment by Arturlang** [comment]
> They wouldnt be hard to kill if fortitude did not give them burn damage resistance, which they are normally weak to. Iirc it was added in one of previous PRs. Any high level vampire can become an immune to stuns bullet sponge. Add the burn resistance and a good laser armor on top and even lasers have trouble dealing enough damage (this is what some players did).
> 
> I believe that with fortitude no longer giving burn resistance like it was originally, there is no way a bloodsucker can become close to unstoppable.

No burn resistance would make fortitude mostly useless, as that's your main bane, though I'd be willing to reduce max brute and burn resist if it's the numbers that's the main issue here, though I'd like more feedback on that

**Comment by Arturlang** [comment]
I'll do the fortitude damage resist changes in a separate PR if it's neccesary

File: modular_zubbers/code/modules/antagonists/bloodsucker/powers/fortitude.dm [hunk 1]
```diff
@@ -1,16 +1,18 @@
 #define FORTITUDE_STUN_IMMUNITY_LEVEL 4
 /datum/action/cooldown/bloodsucker/fortitude
 	name = "Fortitude"
-	desc = "Withstand egregious physical wounds and walk away from attacks that would stun, pierce, and dismember lesser beings."
+	desc = "Withstand egregious physical wounds and walk away from attacks that would stun, pierce, and dismember lesser beings, but will render you unable to heal."
 	button_icon_state = "power_fortitude"
 	power_flags = BP_CONTINUOUS_EFFECT|BP_AM_COSTLESS_UNCONSCIOUS
-	check_flags = NONE
+	check_flags = BP_CANT_USE_IN_TORPOR|BP_CANT_USE_IN_FRENZY
```
Comment (Arturlang) [inline modular_zubbers/code/modules/antagonists/bloodsucker/powers/fortitude.dm:7]: this is re-adding old check_flags that i accidentally removed

**Pull request merged**